empress/recon_vis/recon_viewer.py

```diff
@@ -369,7 +375,7 @@ def _calculate_font_size(num_tip_nodes: int):
     :return the font size for the tips and internal nodes of a tree
     """
     x = (render_settings.START_SIZE - num_tip_nodes) / render_settings.STEP_SIZE
-    return 1.5 * _sigmoid(x)
+    return 3.0 * _sigmoid(x)  # 3.0 is a magic value that can be adjusted
```

Comment:
These magic values should probably go at the top of the file to make it clearer that they exist and how to adjust them.